t/50file-file.t

```diff
@@ -51,4 +51,26 @@ EOT
     });
 };
 
+subtest 'dynamic with default type' => sub {
+    eval q{use CGI; 1}
+        or plan skip_all => 'CGI.pm not found';
+
+    # spawn h2o
+    my $server = spawn_h2o(<< "EOT");
+file.custom-handler:
+  extension: default
+  fastcgi.spawn: "exec \$H2O_ROOT/share/h2o/fastcgi-cgi"
+hosts:
+  default:
+    paths:
+      /robots.txt:
+        file.file: @{[DOC_ROOT]}/noextcgi
+EOT
+    run_with_curl($server, sub {
+        my ($proto, $port, $cmd) = @_;
+        my $resp = `$cmd --silent $proto://127.0.0.1:$port/robots.txt?name=Tobor`;
+        is $resp, "Hello Tobor";
+    });
```

Comment:
The test tests if the custom handler works well with `file.file`.

Could we also have a test case that maps all files under a directory to CGI? To rephrase, I would appreciate it if we could have a test case covering the use-case of `SetHandler cgi-script` in Apache.

```
<Directory "/home/*/public_html/cgi-bin">
    Options ExecCGI
    SetHandler cgi-script
</Directory>
```